### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by frederikbosch**
Updated the following items.
- Installation
- Removed integration
- Removed dead links in inspiration
- Added integer limits
- Added formatting
- Added parsing
- Added json

@sagikazarmark Can you document the things you have worked on? I think that gives best results. Please let me know if you run out of time. Then I can create docs for you work too.

### TODO

- [x] Improve code blocks
- [x] Clean markup

### CODE REVIEW

**Comment by sagikazarmark** [comment]
Will do in the next few days as I will be on holiday.

File: doc/Bitcoin.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+Bitcoin
+=======
+
+If you want to work with the Bitcoin currency, Money has support for this since version 3.0.0. You can construct a
```
Comment (sagikazarmark) [inline doc/Bitcoin.rst]: RTD allows us to maintain separate versions of documentations. I don't think it's necessary to add version here.

File: doc/GettingStarted.rst [hunk 2]
```diff
@@ -11,19 +11,16 @@ All amounts are represented in the smallest unit (eg. cents), so USD 5.00 is wri
...
+------------
 
-You'll need an autoloader. Money is PSR-0 compatible, so if you are using the Symfony2 autoloader, this will do:
+Install the library using composer. Add the following to your composer.json:
+
+.. code-block:: json
+
+{
+    "require": {
+        "mathiasverraes/money": "~3.0"
```
Comment (sagikazarmark) [inline doc/GettingStarted.rst]: I would rather use carret.

Comment (frederikbosch) [inline doc/GettingStarted.rst]: Copied from readme. I believe current standards indicate we should use cli: composer require. And then composer its preference for versioning. I'll update both.

File: doc/Bitcoin.rst (hunk 1 above)
Comment (frederikbosch) [inline doc/Bitcoin.rst]: True, will update.

File: doc/GettingStarted.rst (hunk 2 above)
Comment (sagikazarmark) [inline doc/GettingStarted.rst]: Agreed. Not sure if a version is necessary at all. composer require will install the latest stable.

Comment (frederikbosch) [inline doc/GettingStarted.rst]: Cool, let's do that then.

**Comment by sagikazarmark** [comment]
I am going to finish this during the weekend and review the rest of the open issues/feature request. Then I would say we are ready for an RC.

**Comment by frederikbosch** [comment]
I'd say we merge this one too today.

**Comment by sagikazarmark** [comment]
Agreed, but I would do some reviews and add some content in the eveining.

**Comment by frederikbosch** [comment]
@sagikazarmark Please do. You can merge it in directly. Documentation should become an on-going process.

**Comment by sagikazarmark** [comment]
Agreed. Also, the conflicts must be solved.

**Comment by frederikbosch** [comment]
Yeah, I'll take care of that. Though merging and rebasing is not something I am particularly good at.

**Comment by sagikazarmark** [comment]
Leave it to me then.

**Comment by frederikbosch** [comment]
@sagikazarmark Sure :)

**Comment by sagikazarmark** [comment]
@frederikbosch finally we get this sorted out. :smile:

**Pull request merged**